master/internal/rm/multirm/multirm.go

```diff
@@ -547,3 +547,8 @@ func (m *MultiRMRouter) fanOutRMCommand(f func(rm.ResourceManager) error) error
 	}
 	return nil
 }
+
+// SmallerValueIsHigherPriority returns true if smaller priority values indicate a higher priority level.
+func (m *MultiRMRouter) SmallerValueIsHigherPriority() (bool, error) {
+	return false, nil
```

Comment:
Just to confirm, are we setting this to `false` because multiRM implies the cluster must be all k8s RMs, and k8s RMs use higher value -> higher priority?